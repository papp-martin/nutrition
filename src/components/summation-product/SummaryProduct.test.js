import { shallow } from "enzyme";
import React from "react";
import SummaryProductForTest from './summary-productForTest.component';


let wrapper;
beforeEach(() => {
    const mockProps = {
        summationProduct: {},
        clearProduct: jest.fn(),
        quantityChanging: jest.fn()
    }

    wrapper = shallow(<SummaryProductForTest {...mockProps} />)
})

it('renders SummaryProductForTest without error', () => {
    expect(wrapper).toMatchSnapshot();
})

it('handleChange method checking', () => {
    wrapper.find("[id='gramm-input']").simulate('change', { target: { value: '100' } });
    expect(wrapper.state()).toEqual({ quantity: 100 });
})
