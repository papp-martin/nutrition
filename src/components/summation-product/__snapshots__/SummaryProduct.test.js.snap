// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SummaryProductForTest without error 1`] = `ShallowWrapper {}`;
